test/files/run/type-tag-leak.scala

```diff
@@ -0,0 +1,30 @@
+import scala.reflect.runtime.universe
```

Comment:
To follow the convention, perhaps we could call this `t11767.scala`.

Reply:
That would have helped me. A new PR fails this test, and I was like, where is that test, when was it deleted? Finally realized it was on 2.12 and also recent. However, TIL `--full-history`.